lib/internal/Magento/Framework/Api/Test/Unit/Code/Generator/_files/SampleExtension.txt

```diff
@@ -40,4 +40,23 @@ class ProductExtension extends \Magento\Framework\Api\AbstractSimpleObject imple
         $this->setData('complex_object_attribute', $complexObjectAttribute);
         return $this;
     }
+
+    /**
+     * @return \Magento\Bundle\Api\Data\BundleOptionInterface|null
+     */
+    public function getComplexObjectAttributeWithTypeDeclaration()
+    {
+        return $this->_get('complex_object_attribute_with_type_declaration');
+    }
+
+    /**
+     * @param \Magento\Bundle\Api\Data\BundleOptionInterface
```

Comment:
param annotation should looks like:
* @param \Magento\Bundle\Api\Data\BundleOptionInterface $complexObjectAttributeWithTypeDeclaration

but this is not issue of this PR